src/components/HtxTextBox/HtxTextBox.js

```diff
@@ -9,7 +9,7 @@ const { Paragraph } = Typography;
 
 export class HtxTextBox extends React.Component {
   state = {
-    editing: false,
+    editing: true,
```

Comment:
Do we want to change this from the default? I know that the LabelStudio interface you can specify the textbox to have `editable=True`. If the `editable` field is the same as setting `editing`, then maybe we can make the change at the LabelStudio interface level.

Reply:
@anishnag Initially setting the editing as true, always ensures that the text area is editable on load. Setting the initial value of `editing` to the editable prop makes sense

Reply:
Though editing is not the same as editable, editing is a state that controls when to show the text area and when to show the view area. Editable is used to check if the text area should be shown at all.